src/libs/blueprint/conduit_blueprint_mesh.hpp

```diff
@@ -157,6 +162,10 @@ namespace coordset
         bool CONDUIT_BLUEPRINT_API verify(const conduit::Node &n,
                                           conduit::Node &info);
 
+        //---------------------------------------------------------------------
+        bool CONDUIT_BLUEPRINT_API transform(const conduit::Node &n,
+                                             conduit::Node &dest);
```

Comment:
while it makes sense for verify -- transform w/o a protocol seems too generic.

for these leaf case transforms, what about `to_uniform()`, etc (I think this mirrors our mcarray transforms)

Reply:
The way this is organized actually confuses me a bit because the style in `mcarray` seems to conflict with that used in `mesh`. We can discuss this later; it's difficult to convey in this context.